src/auth/OAUthWithIDTokenValidation.js

```diff
@@ -4,6 +4,9 @@ var Promise = require('bluebird');
 
 var ArgumentError = require('rest-facade').ArgumentError;
 
+var HS256_IGNORE_VALIDATION_MESSAGE =
+  'Validation of `id_token` requires a `clientSecret` when using the HS256 algorithm. To ensure tokens are validated, please switch the signing algorithm to RS256 or provide a `clientSecret` in the constructor.';
```

Comment:
should the back-ticks be double quotes instead or are they rendered differently in the console?

Reply:
I use back-ticks when I'm talking about code things (in this case, they're params or the actual token etc)